.travis/build.sh

```diff
@@ -55,7 +55,7 @@ echo "Docker push with registry $DOCKER_REGISTRY and org $DOCKER_ORG under tag $
 make docker_push
 
 OLD_DOCKER_ORG=$DOCKER_ORG
-export DOCKER_ORG="localhost:5000/strimzici"
+export DOCKER_ORG="strimzici"
```

Comment:
You removed the `localhost:5000` which was AFAIK used by Travis for STs. I do not see it added anywhere. How did you addressed it?

Reply:
Thanks for the question, you can see that we define `DOCKER_REGISTRY="localhost:5000"` for Travis in build.sh file, link - https://github.com/strimzi/strimzi-kafka-operator/blob/9b1afa8b925cad7ba924098e82789378ad174413/.travis/build.sh#L48